homeassistant/components/wiz_light/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""WiZ Light integration."""
+import logging
+
+from pywizlight import wizlight
+from pywizlight.exceptions import WizLightConnectionError, WizLightTimeOutError
+
+from homeassistant.components.wiz_light.light import WizBulb
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["light"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Old way of setting up the wiz_light component."""
+    hass.data[DOMAIN] = {}
+    return True
```

Comment:
```suggestion
```

Reply:
You can drop this completely now in favor of https://github.com/home-assistant/core/pull/44779/files#r757152283